src/rgw/rgw_sal_motr.cc

```diff
@@ -751,21 +769,33 @@ int MotrBucket::link_user(const DoutPrefixProvider* dpp, User* new_user, optiona
   new_bucket.encode(bl);
   std::time_t ctime = ceph::real_clock::to_time_t(new_bucket.creation_time);
   ldpp_dout(dpp, 20) << "got creation time: << " << std::put_time(std::localtime(&ctime), "%F %T") << dendl;
+  string tenant_bucket = info.bucket.tenant;
+  if(tenant_bucket != ""){
+    tenant_bucket += "$"+info.bucket.name;
+  }else{
+    tenant_bucket = info.bucket.name;
+  }
 
   // Insert the user into the user info index.
   string user_info_idx_name = "motr.rgw.user.info." + new_user->get_info().user_id.to_str();
   return store->do_idx_op_by_name(user_info_idx_name,
-                                  M0_IC_PUT, info.bucket.name, bl);
+                                  M0_IC_PUT, tenant_bucket, bl);
 
 }
 
 int MotrBucket::unlink_user(const DoutPrefixProvider* dpp, User* new_user, optional_yield y)
 {
   // Remove the user into the user info index.
   bufferlist bl;
+  string tenant_bucket = info.bucket.tenant;
+  if(tenant_bucket != ""){
+    tenant_bucket += "$"+info.bucket.name;
+  }else{
+    tenant_bucket = info.bucket.name;
+  }
```

Comment:
style: add spaces between `if(` and `}else{` here and everywhere in your changes. Follow the coding style of the existing source code please.

Reply:
Sorry @andriytk these are changes from other PR, just reverted the changes. Please review